streams/src/test/java/org/apache/kafka/streams/integration/StandbyTaskEOSIntegrationTest.java

```diff
@@ -102,17 +119,19 @@ public void surviveWithOneTaskAsStandby() throws ExecutionException, Interrupted
 
             streamInstanceOne.close(Duration.ZERO);
             streamInstanceTwo.close(Duration.ZERO);
+
+            streamInstanceOne.cleanUp();
```

Comment:
Previously, the test was executed only once. Now it's executed twice. Because we use the same `appId` for both runs it seems better to add the cleanup (even if we might get two different temp directories anyway, so it might not be required)